dataproc_jupyter_plugin/tests/test_dataproc.py

```diff
@@ -20,22 +20,14 @@
 async def test_list_clusters(monkeypatch, jp_fetch):
     mocks.patch_mocks(monkeypatch)
 
-    mock_project_id = "credentials-project"
     mock_page_token = "mock-page-token"
-    mock_region_id = "mock-region"
-    mock_page_size = "mock_page_size"
+    mock_page_size = 1
     response = await jp_fetch(
         "dataproc-plugin",
         "clusterList",
         params={"pageSize": mock_page_size, "pageToken": mock_page_token},
     )
     assert response.code == 200
-    payload = json.loads(response.body)
```

Comment:
We still need to parse and validate the response body, which means we should also be mocking the list clusters call within the Dataproc client library.